src/renderer/components/+apps-helm-charts/helm-chart-details.tsx

```diff
@@ -47,6 +47,21 @@ export class HelmChartDetails extends Component<Props> {
     this.chartPromise?.cancel();
   }
 
+  componentDidUpdate(prevProps: Props) {
+    if (this.props.chart !== prevProps.chart) {
+      this.selectedChart = null
+      const { chart: { name, repo, version } } = this.props
+      helmChartsApi.get(repo, name, version).then(result => {
+        this.readme = result.readme
+        this.chartVersions = result.versions
+        this.selectedChart = result.versions[0]
+      },
+      error => {
+        this.error = error;
+      })
+    }
+  }
+
```

Comment:
Can you please revactor your changes a bit in a way that get request won't be duplicated in `componentDidMount` and `componentDidUpdate`?

You could use mobx `autorun` or `reaction` defined inside `componentDidMount` so it will trigger the fetch for both described methods (when `props.chart` will be changed).

You can reference usage of reaction inside `src/renderer/components/+user-management-roles-bindings/role-binding-details.tsx` for example.

Reply:
Hi, thanks for your comment! 
Changed to autorun because "reaction" doesn't work on element init

Reply:
Did you push your changes with autorun? I still see an old code in PR.

Reply:
Yeah,  forget to add in the commit before pushing, fixed.